modules/bedToNrf.cfmod.pl

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env perl
+use warnings;
+use strict;
+use Getopt::Long;
+use FindBin qw($Bin);
+use lib "$FindBin::Bin/../source";
+use CF::Constants;
+use CF::Helpers;
+use File::Copy qw(move);
```

Comment:
I think you can drop this dependency? Not sure that it's being used in any of your new modules here..